bin/.travis/push.sh

```diff
@@ -26,38 +25,32 @@ if [ "$1" = "" ]; then
     exit 1
 fi
 
+if [ "$2" = "" ]; then
+    echo "Argument 2 variable VERSION_FORMAT is not set, format: ezsystems/php. Bailing out !"
+    exit 1
+fi
+
 REMOTE_IMAGE="$1"
+VERSION_FORMAT="$2"
+
 PHP_VERSION=`docker -l error run ez_php:latest php -r "echo PHP_MAJOR_VERSION.'.'.PHP_MINOR_VERSION;"`
-NODE_VERSION=`docker -l error run ez_php:latest-node node -e "console.log(process.versions.node)"` 
+NODE_VERSION=`docker -l error run ez_php:latest-dev node -e "console.log(process.versions.node)"` 
 
 docker images
 docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
 
 ## TAGS
 echo "About to tag remote image '${REMOTE_IMAGE}' with php version '${PHP_VERSION}' and Node '${NODE_VERSION}'"
 
-# "7.0"
-docker tag ez_php:latest "${REMOTE_IMAGE}:${PHP_VERSION}"
-docker tag ez_php:latest-node "${REMOTE_IMAGE}:${PHP_VERSION}-node"
-docker tag ez_php:latest-dev "${REMOTE_IMAGE}:${PHP_VERSION}-dev"
```

Comment:
why was these skipped? I would rather say we should remove them from v1 branch once this is merged and v1 is branched out, or?  (so tags like this will point to latests version format tags)